package.json

```diff
@@ -2,22 +2,20 @@
   "name": "spectacle",
   "version": "5.5.0",
   "description": "ReactJS Powered Presentation Framework",
+  "source": "src/components/index.js",
   "main": "lib/index.js",
   "module": "es/index.js",
```

Comment:
`lib/` and `es/` won't exist with this infrastructure. Let's make sure that we point these to workable, appropriate CJS and ESM rollup bundles.

Reply:
I'll remove these entries for now as we'll work on these bundles in the next PR.